src/librbd/ImageWatcher.cc

```diff
@@ -163,42 +163,45 @@ void ImageWatcher<I>::notify_snap_create(const std::string &snap_name,
 template <typename I>
 void ImageWatcher<I>::notify_snap_rename(const snapid_t &src_snap_id,
 				         const std::string &dst_snap_name,
-                                      Context *on_finish) {
+					 Context *on_finish) {
   assert(m_image_ctx.owner_lock.is_locked());
   assert(m_image_ctx.exclusive_lock &&
          !m_image_ctx.exclusive_lock->is_lock_owner());
 
-  notify_lock_owner(SnapRenamePayload(src_snap_id, dst_snap_name), on_finish);
+  notify_lock_owner(SnapRenamePayload(src_snap_id, dst_snap_name, cls::rbd::UserSnapshotNamespace()), on_finish);
 }
 
 template <typename I>
 void ImageWatcher<I>::notify_snap_remove(const std::string &snap_name,
+					 const cls::rbd::SnapshotNamespace &snap_namespace,
                                          Context *on_finish) {
```

Comment:
@dillaman I'm probably asking it too late, but still. Which is a preferred style for figure brackets for functions - on the same line or on the next line. I see both variants, I usually follow the style which is in the file, but finally got curious. Which one is correct?